lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -165,6 +166,7 @@ class UnusedBytes extends Audit {
       const networkNode = /** @type {NetworkNode} */ (node);
       const originalTransferSize = originalTransferSizes.get(networkNode.record.requestId);
       if (originalTransferSize === undefined) return;
+      networkNode.record.transferSize = originalTransferSize;
       networkNode.record._transferSize = originalTransferSize;
```

Comment:
should we just use one of these? this PR is switching to use the other already. 
Also happy to fix the `_prop` use in a followup to minimize the diff.

Reply:
yeah eliminated `_transferSize`, the dance was necessary for transferSize since the weird way the getter interacts with our simulation but the rest will be followup 👍